addons/website_crm_partner_assign/static/src/js/crm_partner_assign.js

```diff
@@ -250,22 +248,30 @@ publicWidget.registry.crmPartnerAssign = publicWidget.Widget.extend({
      * @private
      * @param {Event} ev
      */
-    _onCalendarIconClick: function (ev) {
-        $(ev.currentTarget).closest('div.date').datetimepicker({
+    _onCalendarInputGroupClick: function (ev) {
```

Comment:
Somehow you have to make both _onCalendarIconClick and _onCalendarInputGroupClick co-exist (probably using a sub method for most of the code ?) until we reach master. Indeed when dealing with stable you have to consider
  * people doing nothing, just pulling code;
  * people pulling code, refresh JS;
  * people pulling code, doing a -u, not refreshing JS;
  * people pulling code, doing a -u, refreshing JS.

Globally you should never remove JS as it may break depending on how people deal with code update and view upgrade.

Cheers,